releasenotes/notes/30261.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: release-notes/v2
+kind: bug-fix
+area: networking
+issue:
+- 30261
+releaseNotes:
+- |
+  **Fixed**  Periodically resolve jwks_uri.
```

Comment:
could you reword this as a new feature and also include a link to the github issue?

Reply:
Ah I thought the:
```
issue:
- 30261
```
Is already providing the information on the issue without providing a link in the release notes.